resources/authgear/templates/en/web/__turbo_error.html

```diff
@@ -0,0 +1,173 @@
+<turbo-stream action="replace" target="error-message">
```

Comment:
This change should be belong to this commit. And the filename should be `turbo_error.html`. It `{{ template }}` `__error.html`.